src/dict/dictionary.cpp

```diff
@@ -235,11 +264,17 @@ void Dictionary::sortTerms(SharedTermList &terms) const
     std::sort(std::begin(*terms), std::end(*terms),
         [] (const SharedTerm lhs, const SharedTerm rhs) -> bool
         {
-            return lhs->clozeBody.size() > rhs->clozeBody.size() ||
-                (
-                    lhs->clozeBody.size() == rhs->clozeBody.size() &&
-                    lhs->score > rhs->score
-                );
+            if(lhs->clozeBody.size() != rhs->clozeBody.size())
+            {
+                return lhs->clozeBody.size() > rhs->clozeBody.size();
+            }
+            bool lhsIsConjugated = lhs->conjugationExplanation.size() > 0;
+            bool rhsIsConjugated = rhs->conjugationExplanation.size() > 0;
+            if(lhsIsConjugated != rhsIsConjugated)
+            {
+                return lhsIsConjugated < rhsIsConjugated;
```

Comment:
The idea is to prioritize exact match terms over conjugated terms, for example if you search 上げ, the noun definition is more likely to be relevant then the masu stem verb definition, or if you search 従って, the entry for "therefore" is more likely to be relevant then 従う -te form.
It can be simplified to `return rhsIsConjugated;`

Reply:
That makes sense. I'd prefer the simplified version since it is looks less like a mistake.